composer.json

```diff
@@ -127,7 +127,11 @@
                 "3248175: Symfony 4.4 event dispatcher parameter order change": "https://www.drupal.org/files/issues/2021-11-08/symfony-4-4-event-dispatcher-parameter-order-change-3248175-2.patch"
             },
             "drupal/socialbase": {
-                "[Internal] Add extra check for the file.": "https://git.drupalcode.org/project/socialbase/-/merge_requests/81.diff"
+                "[Internal] Add extra check for the file.": "https://git.drupalcode.org/project/socialbase/-/merge_requests/81.diff",
+                "YANG-6475 Make join methods for groups re-usable": "https://www.drupal.org/files/issues/2021-12-29/re-use-group-join-methods-3256324-3.patch"
```

Comment:
Those patches should be replaced with dependency from the theme version, once we approve and merge all changes.

Reply:
I'd say that this patch should be removed by someone who will change the theme version which contains this patch.